_sass/common.scss

```diff
@@ -141,3 +141,23 @@ header,
 .invisible {
     display: none !important;
 }
+//login scope
+.fx-acc {
```

Comment:
@sashcy seems this element doesn't exist in oauth page, are you planning to create a separate PR for that to add this element? or is it better to use the existing elements?